the-faseb-journal.csl

```diff
@@ -0,0 +1,126 @@
+<?xml version="1.0" encoding="utf-8"?>
+<style xmlns="http://purl.org/net/xbiblio/csl" version="1.0"
+    default-locale="en-US"
+    class="in-text" 
+    page-range-format="expanded"
+    demote-non-dropping-particle="sort-only"
+    initialize-with=". "
```

Comment:
yes, include them in lines 32 and 38 (of this version of the style) respectively, i.e. as part of the cs:name. 
There is nothing in the specifications that makes what you did wrong, but (almost?) all other styles have these attributes set on the name element and keeping this uniform will make it much easier to maintain styles.